projects/storefrontapp/src/app/app.module.ts

```diff
@@ -65,6 +65,7 @@ if (!environment.production) {
       },
       features: {
         level: '1.3',
+        anonymousConsents: true,
```

Comment:
by default again, allow anonymousConsents to always on?

Reply:
Yes, on by default on our servers and local development. Remember, we don't ship the shell app to clients.